setup_ts.py

```diff
@@ -24,8 +24,8 @@
 from zipfile import ZipFile
 
 
-node_version = 'v16.14.2'
-yarn_version = 'v1.22.10'
+node_version = 'v18.12.1'
+yarn_version = 'v1.22.19'
```

Comment:
I thought there is no yarn any more.

Reply:
haven't written ts for a long time, now do we still have `yarn watch`?

Reply:
> I thought there is no yarn any more.

yarn is still used in webui & jupyter extension.
It can be fully removed, but don't want to do it now.

Reply:
> haven't written ts for a long time, now do we still have `yarn watch`?

It will need a separate watcher like [npm-watch](https://www.npmjs.com/package/npm-watch). Have not added it because myself do not use it.